### ISSUE TITLE
Please publish an updated package

### ISSUE THREAD

**Issue by connor4312** [node-influx/node-influx#136]
We're waiting on the bug fixed in #126 to deploy some new stuff. The PR was merged, but the package version was not updated. It'd be great if node-influx could be bumped!

**Comment by bencevans** [comment]
Just released `4.2.0` (6ba89be2d26e5d486e82ab1e8afaa04b1608bb97). Fyi it's possible to pin dependencies to a particular commit, branch or tag of a git repo... https://docs.npmjs.com/cli/install

If you're actively using the project and given you're contributing :+1:  Would be up for helping maintain? Need more people on the team to even out the workload (review PR's, add support for latest Influx releases etc.).

**Comment by connor4312** [comment]
It's possible to pin dependencies, but that breaks on Cygwin which some of our developers use. npm doesn't _officially_ support Cygwin but it does work except for a few specific cases, one of which is when installing dependencies from git.

We are actively using it, and I'd definitely be up to help maintain it. I was intending to put in another PR or two in the next couple days anyhow 😄

**Comment by bencevans** [comment]
Sweet, I'll add you on. If one other person, other than the committer can review pull-requests before merge, that'd be dandy :smile_cat: 

Org wise, prob need to step up the game. Maybe start using http://www.wsj.com/articles/the-power-of-the-doodle-improve-your-focus-and-memory-1406675744 and setup a gitter channel for ramblings and chattings rather than spamming issues. Any thoughts?

**Comment by connor4312** [comment]
I think a Gitter channel would be a good idea for a general discussion, but I'd prefer issue-related discussion to be kept on that issue. It makes it easier for others to come by and see the status/thoughts regarding the issue, and is a good record-keeper for issues that might not get dealt with immediately (rather than scrolling back forever in history).

**Comment by bencevans** [comment]
Yeah, sorry that was misleading. Meant anything not related to specific issues
:)

On May 5 2016, at 6:50 pm, Connor Peet &lt;notifications@github.com&gt; wrote:  

> I think a Gitter channel would be a good idea for a general discussion, but
I'd prefer issue-related discussion to be kept on that issue. It makes it
easier for others to come by and see the status/thoughts regarding the issue,
and is a good record-keeper for issues that might not get dealt with
immediately (rather than scrolling back forever in history).

>

> —  
You are receiving this because you modified the open/close state.  
Reply to this email directly or [view it on GitHub](https://github.com/node-
influx/node-influx/issues/136#issuecomment-217224450)![](https://github.com/no
tifications/beacon/AAm-R0U_0wbgWN90q15-EsFfdKyu5Ervks5p-i3EgaJpZM4IXfIk.gif)

**Comment by connor4312** [comment]
Yep, sounds good!

**Comment by bencevans** [comment]
Awesome. node-influx org isn't showing up in Gitter atm. Shall try again
soon... might be because i've just updated the memberships etc. on the org O.o

On May 5 2016, at 6:52 pm, Connor Peet &lt;notifications@github.com&gt; wrote:  

> Yep, sounds good!

>

> —  
You are receiving this because you modified the open/close state.  
Reply to this email directly or [view it on GitHub](https://github.com/node-
influx/node-influx/issues/136#issuecomment-217225089)![](https://github.com/no
tifications/beacon/AAm-R5mY9gYICaFZhGINBHKTKeYjsTeYks5p-i5QgaJpZM4IXfIk.gif)
